bindings/rust/s2n-tls/src/raw/connection.rs

```diff
@@ -113,6 +113,13 @@ impl Connection {
         Ok(self)
     }
 
+    /// Reports the remaining nanoseconds before the connection may be safely closed.
+    ///
+    /// If [`shutdown`] is called before this method reports "0", then an error will occur.
+    pub fn remaining_blinding_nanos(&self) -> Result<u64, Error> {
+        unsafe { s2n_connection_get_delay(self.connection.as_ptr()).into_result() }
+    }
```

Comment:
Rust has a `core::time::Duration` struct for durations like this. It makes it really nice not to have to know the units

```suggestion
    pub fn remaining_blinding(&self) -> Result<core::time::Duration, Error> {
        let nanos = unsafe { s2n_connection_get_delay(self.connection.as_ptr()).into_result() }?;
        Ok(core::time::Duration::from_nanos(nanos))
    }
```

Reply:
D'oh, I know Durations exist, and I immediately convert the result of this function into one in the tokio code, but I didn't consider just returning one in the first place xD